### PR TITLE
Move fxa events queries to bigquery-etl

### PR DESCRIPTION
**Pull request by relud**
requires mozilla/bigquery-etl#114

### CODE REVIEW

File: dags/fxa_events.py [hunk 1]
```diff
@@ -23,34 +24,22 @@
...
 
-    fxa_content_events = BigQueryOperator(
+    fxa_content_events = bigquery_etl_query(
         task_id='fxa_content_events',
-        bql='sql/fxa_content_events_v1.sql',
-        destination_dataset_table='telemetry.fxa_content_events_v1${{ds_nodash}}', ## noqa
-        write_disposition='WRITE_TRUNCATE',
-        use_legacy_sql=False,
-        schema_update_options=('ALLOW_FIELD_ADDITION',),
-        bigquery_conn_id="google_cloud_derived_datasets",
```
Comment (jklukas) [inline dags/fxa_events.py]: This conn_id is significant because it has different creds that are allowed to hit the fxa prod dataset.

Comment (relud) [reply dags/fxa_events.py]: it matches the defaults used in [utils.gcp.load_to_bigquery](https://github.com/mozilla/telemetry-airflow/blob/master/dags/utils/gcp.py#L35), [utils.gcp.export_to_parquet](https://github.com/mozilla/telemetry-airflow/blob/master/dags/utils/gcp.py#L278), and [utils.gcp.bigquery_etl_query](https://github.com/mozilla/telemetry-airflow/blob/master/dags/utils/gcp.py#L368)

Comment (jklukas) [reply dags/fxa_events.py]: Very well then!

**Comment by relud** [comment]
i'm gonna wait until monday to merge this

**Pull request merged**